tools/srst2/srst2.xml

```diff
@@ -0,0 +1,355 @@
+
+
+Usage
+=====
+
+Basic Options
+-------------
+
+**Read Type**
+   - Single-end: Single end read file(s) for analysing (--input_se)
+   - Paired-end: Paired end read file(s) for analysing (--input_pe)
+
+**Job Type**
+    - MLST only: Reports Sequence Types
+    - MLST and VFDB: Reports Sequence Types and user can choose one of the built-in Virulence Factor Datebase (VFDB) strains
+    - MLST and custom database: Reports Sequence Types and user can upload their own custom database
+    - VFDB only: Use can choose one of the built-in Virulence Factor Databasse (VFDB) strains
+    - Custom database only: Use can upload their own custom database
+
+**ST definitions for MLST scheme:**
+    - Required if you want to calculate STs (--mlst_definitions)
+
+**Fasta file of MLST alleles:**
+    - Required if you want to calculate STs (--mlst_db)
+
+**Fasta file for gene database:**
+    - Required if you want details of the sequences. The user must provide their own database (--gene_db)
+
+**VFDB strain:**
+    - Required if you want details of the sequences. The use may choose one of the listed strains (--gene_db)
+
+**Read file type:**
+    - fastq
+    - solexa
+    - fasta
+
+**Character(s) separating gene name from allele number in MLST database:**
+    - Required for all MLST job types
+    - Typically either _ or -
+    - The output from getMLST will identify the delimiter.
+
+**Maximum number of mismatches per read for MLST allele calling:**
+    - Required for all MLST job types
+    - For MLST schemas with inserts this number should be set to a high value (recommended: 250)
+
+**Maximum number of mismatches per read for gene allele calling:**
+    - Required for all VDFB or custom database job types
+    - For genes with inserts this number should be set to a high value (recommended: 250).
+
+**Option Type:**
+    - Basic: Includes only the options listed above
+    - Advanced: Includes the options listed below
+
+-------------------------------
+
+Advanced Options
+----------------
+
+**Minimum %coverage cutoff for gene reporting:**
+    - Default is 90 (--min_coverage)
+
+**Maximum %divergence cutoff for gene reporting:**
+    - Default is 10 (--max_divergence)
+
+**Minimum mean depth to flag as dubious allele call:**
+    - Default is 5 (--min_depth)
+
+**Minimum edge depth to flag as dubious allele call:**
+    - Default is 2 (--min_edge_depth)
+
+**Probability of sequencing error:**
+    - Default is 0.01 (--prob_err)
+
+**Stop mapping after this number of reads have been mapped (otherwise map all):**
+    - Default maps all (--stop_after)
+
+**Other arguments to pass to bowtie2:**
+    --other
+
+**Samtools -q parameter:**
+    - Default is 1 (--mapq)
+
+**Samtools -Q parameter:**
+    - Default is 20 (--baseq)
+
+** Acknowledgments **
+Original Author: Mariam Iskander
+
+Jen Cabral
+Philip Mabon
+Mark Iskander
+
+    </help>
+
+    <citations>
+    </citations>
```

Comment:
add the DOI citation of 10.1128/AAC.01310-13

Example on how to use it @ https://wiki.galaxyproject.org/Admin/Tools/ToolConfigSyntax#A.3Ccitations.3E_tag_set